src/GraphQLParser/ParserContext.cs

```diff
@@ -627,14 +646,15 @@ private GraphQLValue ParseInt(bool isConstant)
             };
         }
 
-        private GraphQLInterfaceTypeDefinition ParseInterfaceTypeDefinition()
+        private GraphQLInterfaceTypeDefinition ParseInterfaceTypeDefinition(GraphQLDescription? description)
         {
             var comment = GetComment();
-            int start = _currentToken.Start;
+            int start = description?.Location.Start ?? _currentToken.Start;
```

Comment:
`GraphQLDescription` node has its own location. Why do you set the start of this location to other nodes?

Reply:
Also I think it would be better to move the `GetDescription()` inside such `ParseXXX` methods and remove the parameter. Then the code will be unified with the `GetComment()` method. How do you think?

Reply:
> `GraphQLDescription` node has its own location. Why do you set the start of this location to other nodes?

Unlike comments, the description is a part of a formal definition of the parent node. So I followed the approach with other AST nodes. For example, `GraphQLObjectTypeDefinition` has a name, and so its location is wider than the corresponding `GraphQLName`'s location.

>Also I think it would be better to move the GetDescription() inside such ParseXXX methods and remove the parameter. Then the code will be unified with the GetComment() method. How do you think?

I tried to place the description parsing closer to the specific ParseXxx, but there was a problem: when we encounter a string (description), we don't know what kind of node we are parsing yet. So we have to read past the description node in order to find out the name of the entity and branch the logic to the corresponding ParseXxx method. That was the reason why I refactored `ParseNamedDefinition` into 2 methods: `ParseNamedDefinition` which parses named nodes without description (and thus immediately knows which node we are at), and `ParseNamedDocumentedDefinition` which is called when we encounter a string literal (description).